Docs/README.md

```diff
@@ -1,16 +1,40 @@
 # cicp_inserter
 
 cicp_inserter allows for easy insertion of CICP data into a PNG file.
+CICP is an efficient way to specify explicit color primaries, transfer function.
```

Comment:
I think most people aren't familiar with "color primaries" and "transfer functions".
Think we should just say "CICP is an efficient way to specify explicit color space."?

(We can keep the part with "It is described in...")

Reply:
Good point.  Sounds good.